keyboards/foxlab/time80/readme.md

```diff
@@ -0,0 +1,27 @@
+# time80
+
+![time80](https://i.imgur.com/8HpYStP.jpg)
+
+Time is an intricate custom keyboard kit from Fox Lab, a group of
+Chinese designers focused on making maximizing the potential for
+design and detail in custom keyboards. Featuring an 18-piece design,
+Time is the custom keyboard with the most number of parts in existence (so far).
+
+* Keyboard Maintainer: [Lukas Alexander](https://github.com/lukelex)
+* Hardware Supported: Time 80 PCB
+* Hardware Availability: [Group Buy](https://geekhack.org/index.php?topic=92349.0)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make foxlab/time80:default
+
+Flashing example for this keyboard ([after setting up the bootloadHID flashing environment](https://docs.qmk.fm/#/flashing_bootloadhid))
+
+    make foxlab/time80:default:flash
+
+**Reset Key**: There are no reset switches or pads. Follow this
+[guide](https://beta.docs.qmk.fm/using-qmk/guides/flashing/flashing_bootloadhid)
```

Comment:
```suggestion
[guide](https://beta.docs.qmk.fm/using-qmk/guides/flashing/flashing_bootloadhid)
```

Reply:
I can't spot the difference. What changed?

Reply:
Sorry, bad copy paste. The suggested change has been updated.